server/CGameHandler.h

```diff
@@ -353,7 +357,9 @@ class CGameHandler : public IGameCallback, public CBattleInfoCallback, public En
 	void checkVictoryLossConditions(const std::set<PlayerColor> & playerColors);
 	void checkVictoryLossConditionsForAll();
 
-
+	const std::string complainNoCreatures;
+	const std::string complainNotEnoughCreatures;
+	const std::string complainInvalidSlot;
 };
 

```

Comment:
Lets declare them in CPP file in global scope or do you plan to use them somewhere else outside CGameHandler?

Reply:
These constants are private, so I'm not going to use them somewhere outside CGameHandler.
In other hand, there are some other classes in the cpp file, so...